### PR TITLE
Fix for #43

### PR DESCRIPTION
**Pull request by SeanNaren**

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -140,11 +140,10 @@ def main():
     if args.continue_from:
         print("Loading checkpoint model %s" % args.continue_from)
         package = torch.load(args.continue_from)
-        print(package['epoch'])
         model.load_state_dict(package['state_dict'])
         optimizer.load_state_dict(package['optim_dict'])
-        start_epoch = int(package.get('epoch', 1)) - 1  # Python index start at 0 for training
-        start_iter = int(package.get('iteration', -1)) + 1
+        start_epoch = int(package.get('epoch', None) or -1) - 1  # Python index start at 0 for training
+        start_iter = int(package.get('iteration', None) or -1) + 1
```
Comment (EgorLakomkin) [inline train.py]: Now I think iteration is always 'N/A' as the call to checkpoint method never passes the iteration number. 

torch.save(checkpoint(model, optimizer, args, len(labels), epoch, loss_results=loss_results,
                                  wer_results=wer_results, cer_results=cer_results),file_path)

Comment (SeanNaren) [inline train.py]: This is how an 'epoch' checkpoint should be, i.e we only need to save the duration when saving from the middle of an epoch, like [here](https://github.com/SeanNaren/deepspeech.pytorch/blob/master/train.py#L233)!

Comment (EgorLakomkin) [inline train.py]: Ah, I see. So now the problem is recovering checkpoint saved at the end of the epoch. Maybe iteration should be also saved at the end of the epoch also?

Comment (SeanNaren) [inline train.py]: The default is to start at the next epoch, does it not already do this (I should check)

**Comment by SeanNaren** [comment]
Should be fixed!
